gromacs/testing/nosetester.py

```diff
@@ -0,0 +1,218 @@
+"""
+Nose test running.
```

Comment:
Thanks. Lean is good. 

--
Oliver Beckstein
email: orbeckst@gmail.com


Am Feb 23, 2016 um 9:53 schrieb Jan Domański <notifications@github.com>:

> In gromacs/testing/nosetester.py:
> 
> > @@ -0,0 +1,218 @@
> > +"""
> > +Nose test running.
> No, ripped out and adapted from pandas. I assumed nose will need a custom NoseTester class but maybe that's not needed actually - i'll check it out and get back to you on this
> 
> —
> Reply to this email directly or view it on GitHub.
>